code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -280,7 +281,7 @@
 		M.AdjustStun(-40, 0)
 		M.AdjustKnockdown(-40, 0)
 		M.adjustStaminaLoss(-10, 0)
-		M.adjustToxLoss(-2, 0)
+		M.adjustToxLoss(-2, 0, healtoxinlover)
```

Comment:
```suggestion
		M.adjustToxLoss(-2, 0, TRUE)
```
There is no subtype that should act otherwise and deal damage to slimepeople cultists.